repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -43,6 +43,10 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
           println("Spark context available as 'sc' " +
             s"(master = ${_sc.master}, app id = ${_sc.applicationId}).")
           println("Spark session available as 'spark'.")
+          spark.usingHive match {
+            case true => println("Spark session is using Hive catalog.")
+            case _ => println("Spark session is using in-memory catalog.")
+          }
```

Comment:
So we don't want to disclose on the spark shell starting console whether we are using Hive Catalog or In-memory catalog at all? I can remove it, then.